### DISCUSSION TITLE
Account Flagged and No Response from GitHub Support for over One and a half months

### DISCUSSION THREAD

**Discussion by SHUBHAMSHARMA-0711** [community/community#143268] [Other Feature Feedback, Questions, & Ideas]
Dear GitHub Support Team,

It has been more than 45 days since I raised Ticket #2999278 on 15/09/2024, and I have yet to receive a response.

The issue I’m facing is that my GitHub profile is not visible to anyone, even when accessed via the direct link, which results in a **"404 Page Not Found"** error. As a student, my GitHub account is extremely important to me, as many of my projects are hosted on GitHub and are now inaccessible.

Ticket: #2999278
GitHub Username: SHUBHAMSHARMA0711
Github Profile Link: https://github.com/SHUBHAMSHARMA0711

@github-staff, please look into this matter.

**Comment by ebndev** [comment]
Hi @SHUBHAMSHARMA-0711,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
